### ISSUE TITLE
[$250][Accessibility][Tracking] Add Header Role

### ISSUE THREAD

**Issue by rushatgabhane** [Expensify/App#79241]
## Overview

This issue tracks visual headings that are not announced as headings by screen readers, affecting page structure and navigation.

## Context

Part of the accessibility improvement effort tracked in https://github.com/Expensify/Expensify/issues/575078

## Issues to Fix

- https://github.com/Expensify/App/issues/77253 - Visual headings not announced as such
- https://github.com/Expensify/App/issues/76945 - Visual headings not announced as such
- https://github.com/Expensify/App/issues/76943 - Page has incorrect structure of headings

## Instructions

**Suggested Fix:** Add accessibilityRole="header"

> **Note:** Discuss on Slack if you are unsure whether a sub-issue should be fixed or if you are unsure about the right fix / expected result. Ideally, create a single PR that will fix all the issues listed to minimize review time (more PRs only if necessary).
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022009703206237571310</li>
        <li>Upwork Job ID: 2009703206237571310</li>
        <li>Last Price Increase: 2026-01-09</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @abbasifaizan70</details>

**Comment by Uzaifm127** [comment]
#### 🚫 Duplicated proposal withdrawn by 🤖 ProposalPolice.

**Comment by jjcoffee** [comment]
@abbasifaizan70's [proposal](https://github.com/Expensify/App/issues/79241#issuecomment-3730302689) LGTM!

**Comment by jjcoffee** [comment]
> For issue 3, I'm not sure whether we should solve it or not because it requires to change the heading for correct semantic structure

Hmm yeah I think this may need to be worked on separately as it sounds like it would be a large project across the app, but we could potentially fix it specifically for the T&Cs page that's mentioned in the [issue](https://github.com/Expensify/App/issues/76943).

**Comment by neil-marcellini** [comment]
> [@abbasifaizan70](https://github.com/abbasifaizan70)'s [proposal](https://github.com/Expensify/App/issues/79241#issuecomment-3730302689) LGTM!

I agree, hiring.

**Comment by abbasifaizan70** [comment]
@mallenexpensify offer accepted and waiting for payment approval.

**Comment by mallenexpensify** [comment]
D'oh, sorry I forgot to link the job above @abbasifaizan70 . You're paid, summary updated above, thx!
